tests/Functional/Schema/PostgreSQLSchemaManagerTest.php

```diff
@@ -320,6 +321,36 @@ public function testBooleanDefault(callable $comparatorFactory): void
         self::assertFalse($diff);
     }
 
+    /**
+     * @param callable(AbstractSchemaManager):Comparator $comparatorFactory
+     *
+     * @dataProvider \Doctrine\DBAL\Tests\Functional\Schema\ComparatorTestUtils::comparatorProvider
+     */
+    public function testGeneratedColumn(callable $comparatorFactory): void
+    {
+        $wrappedConnection = $this->connection->getWrappedConnection();
+        assert($wrappedConnection instanceof ServerInfoAwareConnection);
```

Comment:
I know some people add this because it helps phpstan or their ide code completion to not complain if later you use a method not present in the interface  so I admitted if was the case here

Reply:
as I didn't get any answer I've removed it

Reply:
@derrabus  any chance for it to get merged as a christmas present ? :santa:  

I think on my side to have adressed all your comments and the tests are as green as a christmas tree :)

Reply:
@derrabus sorry to bother you, I know what it's like to be pinged, but is there something blocking or that I can do to ease the burden of review ?

I've found some other (more blocking) bug related to generated column and I would like to have this PR merged before pushing anything more :)

Reply:
Friendly reminder that  this PR is still an issue today :)